dev/restinio/chunked_input_info.hpp

```diff
@@ -101,6 +129,17 @@ class chunk_info_t
 
 		return make_string_view_nonchecked( full_body );
 	}
+
+	/**
+	 * @brief Get a list of chunk extension's params.
+	 *
```

Comment:
It will be great to add a short example of usage of this method with checking the returned value against `nullptr`.